socorro/external/es/super_search_fields.py

```diff
@@ -1053,6 +1053,9 @@ def apply_schema_properties(fields, schema):
         "query_type": "enum",
         "storage_mapping": {"type": "string"},
     },
+    "crash_inconsistencies": keyword_field(
+        "crash_inconsistencies",
```

Comment:
Virtually all other invocations of `keyword_field` pass the argument as a keyword parameter (except for one), so I suggest we do the same here for consistency:
```suggestion
        name="crash_inconsistencies",
```